fix: join command security schemas fix (#1439)
Redocly/redocly-cli: .changeset/little-camels-fold.md

---
"@redocly/cli": patch
---

> Fix an issue when using `join` command resulted in generating incorrect schema, if security schema used logical AND for security requirements.

Reviewer: ```suggestion
If the security schema uses logical AND for security requirements, the `join` command generates the correct schema, as expected.
```
